src/q5/stat-FSAB1105/exam/2017/Janvier/All/stat-FSAB1105-exam-2017-Janvier-All.tex

```diff
@@ -227,6 +227,8 @@ \section{(4 points)}
 Si $\alpha=0.05$, 
 $$t=2,23  > t_{18; 0,05}=1,734$$
 On rejette $H_{0}$.
+La p-valeur est la valeur $p=P(T>t=2.23)$ où $T \sim t_{n_a+n_b-2}$. Cette valeur n'est pas dans les tables néanmoins l'on trouve $P(T>2.101)=0.025$ donc on peut conclure que $p<0.025$ (et $p>0.01$ par le même raisonnement). L'on pourrait également le montrer graphiquement.
```

Comment:
There is the `\Pr` command for probability

Reply:
would have to change it for almost every question and on other exams too so maybe raise an issue for that ?

Reply:
You don't have to change anything, I just wanted to let you know about it. You can open an issue if you plan to make the change